metrics/prometheus/prometheus.go

```diff
@@ -121,8 +121,8 @@ func CreatePrometheusMetrics(cfg config.PrometheusMetrics) *PrometheusMetrics {
 			),
 		},
 		Connections: &PrometheusConnectionMetrics{
-			ConnectionsClosed: newSingleCounter(cfg, registry, ConnOpenedKey, "Count the number of closed connections"),
-			ConnectionsOpened: newSingleCounter(cfg, registry, ConnClosedKey, "Count the number of open connections"),
+			ConnectionsClosed: newSingleCounter(cfg, registry, ConnClosedKey, "Count the number of closed connections"),
+			ConnectionsOpened: newSingleCounter(cfg, registry, ConnOpenedKey, "Count the number of open connections"),
```

Comment:
Is it possible to add a unit test? ... as to avoid repeating this mistake in the future.